Dockerfile

```diff
@@ -62,6 +61,13 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
+# Install lxc
+RUN curl -sSL https://linuxcontainers.org/downloads/lxc/lxc-1.0.7.tar.gz | tar -v -C /usr/src/ -xz
```

Comment:
Wouldn't this be slightly easier to change versions if we used `tar`'s `--strip-components` so the version number is only hard coded in one spot?